### PR TITLE
Remove django-compressor

### PR DESCRIPTION
**Pull request by rlucioni**
We no longer use this tool. Counterpart to https://github.com/edx/configuration/pull/3267.

@schenedx @mikedikan please review.

### CODE REVIEW

**Comment by schenedx** [comment]
@rlucioni Are you sure the existing django-rest-framework does not use compressor?
If so, 👍

**Comment by rlucioni** [comment]
@schenedx yup, DRF doesn't require compressor.

File: Makefile [hunk 1]
```diff
@@ -30,7 +30,7 @@ clean:
 	coverage erase
 
 requirements:
-	pip install -qr requirements/local.txt --exists-action w
```
Comment (mikedikan) [inline Makefile]: Can you explain these changes to silence the output and the wipe action if a package exists?

Comment (rlucioni) [inline Makefile]: I'm getting rid of both. Having log output is helpful: we want to see what's being installed (or not installed), especially when trying to debug something. Specifying an [exists-action](https://pip.pypa.io/en/stable/reference/pip/#exists-action-option) only appears to be relevant when installing packages using a [VCS system](https://pip.pypa.io/en/stable/reference/pip_install/#vcs-support) (e.g., git), something we're not doing in this project (we install from PyPI).

**Comment by mikedikan** [comment]
👍

**Pull request merged**